src/containers/Movie/MovieInfo/component.js

```diff
@@ -38,6 +42,10 @@ const MovieInfo = ({ info, genres, loading }) => {
               {new Date(releaseDate || firstAirDate).getFullYear()}
               )
             </span>
+            {' '}
+            <Lists />
+            {' '}
```

Comment:
spaces is not best option for creating offsets

Reply:
@VladWithCookies IMHO, in this case, it's a simpler solution that doesn't need additional DOM elements